manifests/init.pp

```diff
@@ -145,6 +148,10 @@
   # enable Varnish service
   include varnish::service
 
+  if($add_hitch) {
+    include varnish::hitch
```

Comment:
this should probably be contain so the class doesn't float around in the catalog:
```suggestion
    contain varnish::hitch
```